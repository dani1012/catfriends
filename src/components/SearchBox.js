import React from 'react';

const SearchBox = ({searchChange})=> {

	return (
	 <div className = 'pa3'>
	   <input 
	       aria-label = 'Search cats'
	       className = 'pa3 ba b--green bg-lightest-blue'
		   type = 'search' 
		   placeholder = 'search cats by name'
		   onChange = {searchChange}
	   />
	 </div>
	)
}



export default SearchBox;