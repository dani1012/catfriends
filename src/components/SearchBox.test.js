import React from 'react';
import SearchBox from './SearchBox';
import Enzyme from 'enzyme';
import {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';


Enzyme.configure({ adapter: new Adapter() });



it ('expects to render Header component', () => {
	const wrapper = shallow(<SearchBox/>);
	expect(toJson(wrapper)).toMatchSnapshot();
})