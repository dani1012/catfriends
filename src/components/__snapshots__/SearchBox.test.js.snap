// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Header component 1`] = `
<div
  className="pa3"
>
  <input
    aria-label="Search cats"
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="search cats by name"
    type="search"
  />
</div>
`;
